### DISCUSSION TITLE
I have an issue to get password from modifies_1_Action to modifies_3_Action but password and othrs in modifies_4_Action (more). If you have any suggestion, it will be nice. basically I like this workflow very much. I appreciate for your time and attetion. Best regards,

### DISCUSSION THREAD

**Discussion by archiei** [blacs30/bitwarden-alfred-workflow#167] [Q&A]

**Comment by blacs30** [comment]
Hi @archiei ,
I don't fully understand the question. Can you post your current configuration for the modifier keys and then how you'd like to have it working. Thanks

**Reply by archiei** [reply]
Dear @blacs30, thank you very much for your kind attention.
I simply did not type "enter key" after modifier.
I have a one question, when I start bitwarden workflow, it will be nice if the workflow open url then get the password without leaving the workflow. are there any way to achieve this feature?  I am attaching my modifier keys data.
Best reards, 
![230903 bitwarden worklow modifiess setting](https://github.com/blacs30/bitwarden-alfred-workflow/assets/10954762/ec0bbb6c-ca44-44c2-aa47-10ee1159c736)

@archiei

**Reply by blacs30** [reply]
@archiei sorry for my late reply. I haven't looked much into it but to achieve what you asked for (auto filling in websites, correct?) but I think the better way would be to use a browser plugin.

**Comment by archiei** [comment]
https://github.com/blacs30
Thank you very much for your kind attention and advise.

**Comment by archiei** [comment]
Thank you very much for you kind advise.

Best reagards,

archiei



On Nov 18, 2023, at 6:24, Claas Lisowski ***@***.***> wrote:


@archiei <https://github.com/archiei> sorry for my late reply. I haven't looked much into it but to achieve what you asked for (auto filling in websites, correct?) but I think the better way would be to use a browser plugin.

**Comment by archiei** [comment]
Dear Claas-san,

I could not open alfred5 after the OS clean installation.
Will you help me on the issue?

Best regards,

Archiei

**Reply by blacs30** [reply]
Dear Archiei-san,

if it is really about Alfred 5 and not the Bitwarden workflow then I think the [Alfred forum](https://www.alfredforum.com/) is a better place to search or ask for help.

Kind regards,
Claas

**Comment by archiei** [comment]
Dear Class-san,

Thank you very much for your quick advise.
I will do so.

Best regards,

Archiei
